.github/workflows/base.yml

```
@@ -28,42 +28,96 @@ jobs:
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Build
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./docker/${{ matrix.dockerfile }}
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.metadata.outputs.tags }}-base-py${{ matrix.python_version }}-cu${{ matrix.cuda }}-${{ matrix.pytorch }}${{ matrix.axolotl_extras != '' && '-' || '' }}${{ matrix.axolotl_extras }}
```

actionlint (1.7.7)

property "axolotl_extras" is not defined in object type {cuda: number; cuda_version: string; cudnn_version: string; dockerfile: string; python_version: number; pytorch: string; torch_cuda_arch_list: string}

(expression)